website/routes.py

```diff
@@ -324,6 +332,16 @@ def make_url_map(app):
             json_renderer,
         ),
 
+        Rule(
+            [
+                '/project/<pid>/citation/<style>/',
+                '/project/<pid>/node/<nid>/citation/<style>/',
```

Comment:
Short answer: I lean towards /citations/style/

Long answer: If it will never be a collection, then it can be singular. However, if you are ever going to do something related to a collection, then you should keep it plural. /citations/ may not return anything on its own yet, but if you were to think of citations as the greater collection of anything that could be narrowed down by selecting a style, then it would make more sense to keep it /citations/style/. So, for example, /citations/apa/ would return the APA style citation. This implies a collection even if none really exists internally, you're just returning a different view of the same data. But from an external user standpoint, there are a collection of citations from which you are grabbing the representation of one.